pages/about.mdx

```diff
@@ -1,7 +1,10 @@
 import Layout from "../components/layout";
+import Footer from "../components/footer";
 
 export default (children) => (
-    <Layout title="About" subtitle="Fiveable Open Source Initiative" {...children} />
+    <div>
+        <Layout title="About" subtitle="Fiveable Open Source Initiative" {...children} />
```

Comment:
was this done by the linter?

Reply:
Nah, originally the footer was imported in this file before we added it to layout.js, and for me to use it with this Layout tag I had to wrap the whole thing in a div. We can just leave this tbh, incase we need it in the future.